project-base/storefront/components/Pages/Search/SearchProducts.tsx

```diff
@@ -0,0 +1,95 @@
+import { SearchProductsContent } from './SearchProductsContent';
+import { useSearchProductsData } from './helpers';
+import { FilterIcon } from 'components/Basic/Icon/IconsSvg';
+import { FilterPanel } from 'components/Blocks/Product/Filter/FilterPanel';
+import { SortingBar } from 'components/Blocks/SortingBar/SortingBar';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import { ProductOrderingModeEnumApi } from 'graphql/generated';
+import { getInternationalizedStaticUrls } from 'helpers/getInternationalizedStaticUrls';
+import useTranslation from 'next-translate/useTranslation';
+import dynamic from 'next/dynamic';
+import { useRef, useState } from 'react';
+import { twJoin } from 'tailwind-merge';
+
+const Overlay = dynamic(() => import('components/Basic/Overlay/Overlay').then((component) => component.Overlay));
+
+export const SearchProducts: FC = () => {
+    const { t } = useTranslation();
+    const paginationScrollTargetRef = useRef<HTMLDivElement>(null);
+    const { url } = useDomainConfig();
+    const [searchUrl] = getInternationalizedStaticUrls(['/search'], url);
+    const [isPanelOpen, setIsPanelOpen] = useState(false);
+
+    const { searchProductsData, fetching, loadMoreFetching } = useSearchProductsData();
```

Comment:
Since you have already renamed the props (with which I agree), could you also rename the constants returned from this hook?

Reply:
I am not sure if I understand. Can you explain exactly what should be those values be replaced with?

Reply:
You have renamed the props from `fetching` to `isFetching`, same for `loadMoreFetching`. My question is if you could make this one more step and rename the returned values from  `useSearchProductsData`

Reply:
I see. Sure! Refactored.